### PR TITLE
Added Russian language support

### PR DESCRIPTION
**Pull request by udachin26**
* [Update translation.yml](https://github.com/FactionMaster/FactionMaster/commit/05bd28c719822704887c17fe6f588c0ff1d6a65d)
* create ru_RU.yml
* [Update ConfigManager.php](https://github.com/FactionMaster/FactionMaster/commit/cc4350671a2f5028713c68fe95effd528864e553)

### CODE REVIEW

File: resources/lang/ru_RU.yml [hunk 1]
```diff
@@ -0,0 +1,271 @@
+# Translate by @udachin26
+# НЕ ТРОГАТЬ!
+file-version: 1
+# НЕ ТРОГАТЬ!
+
+
```
Comment (ShockedPlot7560) [inline resources/lang/ru_RU.yml:5]: An extra space

File: resources/lang/ru_RU.yml [hunk 2]
```diff
@@ -0,0 +1,271 @@
+# Translate by @udachin26
```
Comment (ShockedPlot7560) [inline resources/lang/ru_RU.yml]: Useless line, you will already be thanked in the README etc

File: resources/lang/ru_RU.yml [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+MANAGE_ALLIANCE_MAIN_TITLE: "Управление союзом"
+MANAGE_ALLIANCE_TITLE: "Управление союзом - {{name}}"
+CHANGE_DESCRIPTION_TITLE: "Управление описанием"
+CHANGE_DESCRIPTION_INPUT_CONTENT: "Введите описание здесь"
+CHANGE_MESSAGE_TITLE: "Управление сообщением фракции"
+CHANGE_MESSAGE_INPUT_CONTENT: "Введите сообщение здесь"
+CHANGE_PERMISSION_TITLE: "Управление разрешениями"
+CHANGE_VISIBILITY_TITLE: "Изменить видимость"
+CHANGE_VISIBILITY_STEP: "Выбрать видимость"
+CHANGE_VISIBILITY_EXPLICATION: "§n§7Public :§r\n  The faction will be free to all and everyone can come and go as they please\n \n§n§7Private :§r\n  The faction will be closed to invitations and players will have to be invited by a member\n \n§n§7By invitation :§r\n  Players will be able to send invitations to the faction and the faction will be able to invite players"
```
Comment (ShockedPlot7560) [inline resources/lang/ru_RU.yml]: Any reason why this line is not translated?

File: resources/lang/ru_RU.yml [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+TOP_FACTION_TITLE: "Топ фракции"
+TOP_FACTION_LINE: "§r §6> §7[§e{{rank}}§7]§f {{factionName}}  §o§7Уровень {{level}} \n"
+LEADERBOARD_TITLE: "Таблица лидеров"
+
+CONFIRMATION_TITLE_DELETE_FACTION: "Подтверждение удаления - {{factionName}}"
+CONFIRMATION_CONTENT_DELETE_FACTION: "§fВы уверены, что хотите удалить свою фракцию? Ты не сможешь вернуться"
+CONFIRMATION_TITLE_LEAVE_FACTION: "Подтверждение выхода - {{factionName}}"
+CONFIRMATION_CONTENT_LEAVE_FACTION: "Вы уверены, что хотите покинуть эту фракцию? Ты не сможешь вернуться"
+CONFIRMATION_TITLE_TRANSFER_PROPERTY: "Подтверждение передачи права собственности"
+CONFIRMATION_CONTENT_TRANSFER_PROPERTY: "§fВы уверены, что хотите передать право собственности на эту фракцию? Вы не сможете вернуться, и вас назначат Co-Leader"
```
Comment (ShockedPlot7560) [inline resources/lang/ru_RU.yml]: No equivalent to "Co-Leader", "Recruit" etc  for Russian?

File: resources/lang/ru_RU.yml [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+MAX_HOME_REACH: " §c>> §4Вы достигли предела домов"
+MAX_CLAIM_REACH: " §c>> §4Вы достигли предела claim, доступных на этом сервере"
+MAX_ALLY_REACH: " §c>> §4Вы достигли лимита союзов"
+MAX_ALLY_REACH_OTHER: " §c>> §4Запрашиваемая фракция уже достигла предела своих союзов"
+MAX_MIN_REACH_NAME: " §c>> §4Вы должны войти в название фракции между {{min}} и {{max}}"
+
+RECRUIT_RANK_NAME: "Recruit"
+MEMBER_RANK_NAME: "Member"
+COOWNER_RANK_NAME: "Co-Leader"
+OWNER_RANK_NAME: "Leader"
```
Comment (ShockedPlot7560) [inline resources/lang/ru_RU.yml]: Same question as above, normal that these are not translated?

**Review by ShockedPlot7560 — CHANGES_REQUESTED** [review]

File: resources/lang/ru_RU.yml (hunk 3 above)
Comment (udachin26) [reply resources/lang/ru_RU.yml]: human factor

File: resources/lang/ru_RU.yml (hunk 4 above)
Comment (udachin26) [reply resources/lang/ru_RU.yml]: Ru have

File: resources/lang/ru_RU.yml (hunk 1 above)
Comment (udachin26) [reply resources/lang/ru_RU.yml:5]: Spaces are the same as in en yml

Comment (ShockedPlot7560) [reply resources/lang/ru_RU.yml:5]: As much for me

**Pull request merged**